### PR TITLE
Round Time/Latency view like Dev Tools

### PR DESCRIPTION
**Pull request by ericduran**
Stop hard-coding "ms" in the timing data and switch to using a badly name filter.

I need more filters, I was sloppy the 1st time around.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -152,7 +152,7 @@ <h2 class="no-support">Your browser doesn't support Drag and Drop, or some other
                         <td class="status-column" title="{{entry.status}} {{entry.statusText}}"><div>{{entry.status}}<div class="network-cell-subtitle">{{entry.statusText}}</div></div></td>
                         <td class="type-column" title="{{entry.mimeType}}"><div>{{entry.mimeType}}</div></td>
                         <td class="size-column"><div title="{{entry.size}}">{{entry.size}}<div class="network-cell-subtitle" title="{{entry.contentSize}}">{{entry.contentSize}}</div></div></td>
-                        <td class="time-column"><div title="{{entry.time}}">{{entry.time}}<div class="network-cell-subtitle"  title="{{entry.latency}}">{{entry.latency}}</div></div></td>
+                        <td class="time-column"><div title="{{entry.time}}">{{entry.time | roundMS}}<div class="network-cell-subtitle"  title="{{entry.latency}}">{{entry.latency | roundMS}}</div></div></td>
```
Comment (cvan) [inline index.html]: do you want to round it in the `title` too? it seems like that's how Chrome does it.

Comment (cvan) [inline index.html]: what about [here](https://github.com/ericduran/chromeHAR/blob/532403d/index.html#L145) and [here](https://github.com/ericduran/chromeHAR/blob/532403d/index.html#L162)?

**Comment by ericduran** [comment]
Ha, this is why I made the PR and didn't merge it. Figure you'll have a look at it :-)

**Comment by ericduran** [comment]
Fixed it everyone including a couple of extra places.

**Pull request merged**